src/app/test-record/notes/notes.component.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { NotesComponent } from './notes.component';
+import { SharedModule } from '@app/shared/shared.module';
+import { TestType } from '@app/models/test.type';
+import {CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
+
+describe('NotesComponent', () => {
+  let component: NotesComponent;
+  let fixture: ComponentFixture<NotesComponent>;
+  const testType = {} as TestType;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [SharedModule],
+      declarations: [NotesComponent],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(NotesComponent);
+    component = fixture.componentInstance;
+    component.testType = testType;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+    expect(fixture).toMatchSnapshot();
```

Comment:
Need to a test that checks that edit-note component is present when editMode is true and not present with it is false but test-additionalNotesRecorded is present. Also i would not use an id, it is best using a data selector, this then can also be used for e2e tests as selector.
Will have to check with the tester for the data-selector attr but it is likely that it will be what is used